pkg/lwip/include/arch/sys_arch.h

```diff
@@ -125,13 +125,14 @@ void sys_lock_tcpip_core(void);
 void sys_unlock_tcpip_core(void);
 #define UNLOCK_TCPIP_CORE()        sys_unlock_tcpip_core()
 /** @} */
-
 #ifdef MODULE_RANDOM
 /**
  * @brief   Use `random_uint32()` to generate random numbers, if available
  */
+#if MBEDTLS_ENABLED == 0
```

Comment:
This seems strange - it is set to the same value in lwipopts? Why is this needed?

Reply:
I added (LWIP_RAND) in lwipopts in order to be recognized in `build/pkg/lwip/src/core/dns.c:103`
But it is already defined in `pkg/lwip/include/arch/sys_arch.h:133`
=> This will create a redefiniton, therefore I added `#if MBEDTLS_ENABLED == 0`

Reply:
But you define it to the same thing there: `random_uint32()`. Why not just keep this and remove your extra definition? Make sure the random module is enabled and things should be fine.